modules/@apostrophecms/translation/index.js

```diff
@@ -1,17 +1,173 @@
 module.exports = {
   extend: '@apostrophecms/module',
   options: {
-    enabled: true
+    enabled: true,
+    alias: 'translation'
   },
   init(self) {
-    self.providers = [ { name: 'fakeProvider' } ];
+    self.providers = [];
 
     self.enableBrowserData();
   },
+  handlers(self) {
+    return {
+      '@apostrophecms/doc-type:beforeLocalize': {
+        // Translate the document using the first available provider.
+        async translate(req, doc, source, target, options) {
+          if (!req.query.aposTranslateTargets?.includes(target)) {
+            return;
+          }
+          // We don't support multiple providers yet, so just use the first one
+          // when no provider is specified.
+          const name = self.getProvider(req.query.aposTranslateProvider)?.name;
```

Comment:
Shouldn't we validate query params using `launder`?
Even if there are passed by ourself in the vue app.